packages/date/src/Date.js

```diff
@@ -121,6 +171,7 @@ const AvDate = ({
         data-testid={`date-input-group-${name}`}
       >
         <SingleDatePicker
+          renderMonthElement={renderMonthElement}
```

Comment:
`renderMonthElement` is placed above `{...datePickerProps}` just in case people are already passing in `datePickerProps.renderMonthElement` on their own.